package.json

```diff
@@ -34,7 +34,7 @@
   "dependencies": {
     "axios": "^0.19.0",
     "form-data": "^2.3.3",
-    "websocket": "^1.0.28"
+    "websocket": "^1.0.29"
```

Comment:
I think this happened when you installed this manually inside the project directory. This is fine because it's only a minor version change but generally be careful about this so we don't create breaking changes